app/containers/TransactionsListPage/index.js

```diff
@@ -90,6 +91,7 @@ export class TransactionsListPage extends React.PureComponent {
           messages={messages}
           link="/blockexplorer/transactions"
           onPageChange={this.props.onPageChange}
+          tableMinWidth="1024px"
```

Comment:
One idea - please move all `tableMinWidth` definitions/values to a config file. Then we'll be able to easily change values for all use cases in one place.